Teatoller/reservations
# #002 Project Set-up

#### What does this PR do?
#### Description of Task to be completed?
- [x] set up project file
#### How should this be manually tested?
1. Create a folder on your machine `mkdir myproject`
2. `cd myproject`
3. Run `git clone https://github.com/Teatoller/reservations.git`
4. cd `reservations`
5. open **reservations** in your chosen editor. For **vscode** enter `code .` on terminal and press enter.
6. `git checkout ft-project-setup-001`
7. Run `yarn install`
 
8. Run `yarn start`  the app in the development mode to open [http://localhost:3000](http://localhost:3000) to view it in the browser.
9. Check the display as in **screenshot: 1**
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#### Screenshots (if appropriate)
**Screenshot:1** Reservations Landing Page
![rsvp](https://user-images.githubusercontent.com/39020475/70523277-bc15c400-1b53-11ea-97b6-423fe62784c7.png)

#### Questions:
